src/ui/public/config/__tests__/config.js

```diff
@@ -25,6 +25,28 @@ describe('config component', function () {
       expect(config.get('obscureProperty')).to.be('default');
     });
 
+    it('honors the default parameter for unset options that are exported', () => {
+      if (config.isDefault('dateFormat')) {
```

Comment:
Maybe I'm misunderstanding something about the tests, but would this run for both the `if` and the `else` cases?

Reply:
The `else` block was to handle the scenario where another test didn't properly unset it. I went ahead and changed it to expect it to be unset and go from there, which simplifies the test and I think clarifies its purpose.